src/index.json

```diff
@@ -1394,7 +1394,7 @@
                     "summary": "Additional commands for working with SAP HanaOnAzure instances.",
                     "version": "0.3.3"
                 },
-                "sha256Digest": "12f0f12d9d1f525b2f0cf34b05bfedb5eef87f5b554e80cf3689265726245f25"
+                "sha256Digest": "d1c76239372837386535ecaf226dd0677014e2f2520c1329dbd5fda456000d36"
```

Comment:
What happened to the last wheel that was hosted? and if changes were made, why is the version not bumped????

Reply:
We do not bump our versions every time there is a change. So this is just a minor change, I just updated the .whl file. Let me know if this is okay.

@PakDLiu  have we done this before?

Reply:
@AprilZhaoWY Even minor changes should bump the patch version.

Reply:
@williexu Got it, will make the change in our code and then update this PR

Reply:
Thanks 👍

Reply:
@williexu  Updated! Please take a look